src/client/sandboxes/code-instrumentation/code-instrumentation.js

```diff
@@ -0,0 +1,41 @@
+import PropertiesInstrumentation from './properties/properties';
+import LocationInstrumentation from './location';
+import MethodsInstrumentation from './methods';
+import EventEmitter from '../../utils/event-emitter';
+import { getAttributeProperty } from './properties/attributes';
+import { PROCESS_SCRIPT_METH_NAME, process as processScript } from '../../../processing/js/index';
+
+class CodeInstrumentation {
+    constructor () {
+        this.BODY_CONTENT_CHANGED = 'bodyContentChanged';
+        this.eventEmitter         = new EventEmitter();
+
+        PropertiesInstrumentation.on(PropertiesInstrumentation.BODY_CONTENT_CHANGED,
+            (el) => this.eventEmitter.emit(this.BODY_CONTENT_CHANGED, el));
+    }
+
+    on (event, handler) {
+        return this.eventEmitter.on(event, handler);
+    }
+
+    getAttributesProperty (el) {
+        return getAttributeProperty(el);
+    }
+
+    getOriginalErrorHandler (window) {
+        return PropertiesInstrumentation.getOriginalErrorHandler(window);
+    }
+
+    init (window, document) {
```

Comment:
If we have class we shouldn't have `init` method. All initialization should be done within ctor.